### DISCUSSION TITLE
LIFNode' object has no attribute 'spike'?

### DISCUSSION THREAD

**Discussion by xiaoxiaochaochao** [fangwei123456/SpikingFlow#188] [Q&A] [answered]
Hello,
I met the problem "LIFNode' object has no attribute 'spike'". Could you tell me how solve this problem.

**Comment by fangwei123456** [comment, marked as answer]
Hi, `spike` is removed and no longer used as the attribute as the neuron. But it is still the outputs of the neuron.

**Reply by Redamancy-520** [reply]
So how to solve the problem?

**Reply by fangwei123456** [reply]
You can use hook function to get it. Here is an example of using hook to get firing time: https://github.com/fangwei123456/spikingjelly/discussions/189#discussioncomment-2449106
